node/sc/bridge_manager_test.go

```diff
@@ -153,8 +153,16 @@ func TestBridgeManager(t *testing.T) {
 	sim.Commit() // block
 
 	// Register tokens on the bridge
-	bridge.RegisterToken(&bind.TransactOpts{From: auth2.From, Signer: auth2.Signer, GasLimit: testGasLimit}, tokenAddr, tokenAddr)
-	bridge.RegisterToken(&bind.TransactOpts{From: auth2.From, Signer: auth2.Signer, GasLimit: testGasLimit}, nftAddr, nftAddr)
+	rn, err := bridge.RequestNonce(nil)
+	if err != nil {
```

Comment:
```suggestion
	assert.NoError(t, err)
```

Reply:
I think we need to stop with the fatal error if an error occurs.